### PR TITLE
Adding C# Projections for PushNotifications

### PR DESCRIPTION
**Pull request by loneursid**
Adding a project that creates the C# projections for PushNotifications.

### CODE REVIEW

**Comment by pmpurifoy** [comment]
Look at this PR to find how to include the C# projections in the pipeline. https://github.com/microsoft/WindowsAppSDK/pull/1208/files

I believe you can copy what Howard did with the projection binaries (pdb, dll) and the projection xml (example: Microsoft.Windows.AppLifecycle.Projection.xml).

File: build/CopyFilesToStagingDir.ps1 [hunk 1]
```diff
@@ -116,6 +116,7 @@ PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependenc
 PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
```
Comment (jonwis) [inline build/CopyFilesToStagingDir.ps1:118]: @EHO-makai, @Scottj1s, @BenJKuhn ... do we think that long-term we want lots of these DLLs? Or can they be merged?

**Review by jonwis — APPROVED** [review]

**Pull request merged**